esp/services/esdl_svc_engine/esdl_monitor.cpp

```diff
@@ -518,6 +518,45 @@ class CEsdlMonitor : implements IEsdlMonitor, public CInterface, implements IEsd
                     bindingtree->setProp("@port", port);
                     servicetree->setProp("@name", serviceName.str());
                     servicetree->setProp("@type", "DynamicESDL");
+
+                    // Copy properties from the procpt into the bindingtree for this dynamic service
+                    // Identify the secmgr for the binding template
+                    StringBuffer authMethod;
+                    bindingtree->getProp("Authenticate/@method", authMethod);
+                    xpath.setf("SecurityManagers/SecurityManager[@name='%s']", authMethod.str());
+                    IPropertyTree* secmgr = procpt->queryPropTree(xpath.str());
+                    if (secmgr)
+                    {
+                        // Copy over the dynamic service specific resource map entries
+                        xpath.setf("*/resource_map/%s", serviceName.str());
+                        IPropertyTree* resourceMap = secmgr->queryPropTree(xpath.str());
+                        IPropertyTree* bindingAuthenticate = bindingtree->queryPropTree("Authenticate");
+                        if (resourceMap && bindingAuthenticate)
+                        {
+                            mergePTree(bindingAuthenticate, resourceMap);
+                        }
+                    }
+
+                    // Copy over any dynamic service-specific binding configuration
+                    xpath.setf("applicationBindingInfo/%s", serviceName.str());
+                    IPropertyTree* appBindingInfo = procpt->queryPropTree(xpath.str());
+                    if (appBindingInfo)
```

Comment:
See above about how this makes the top level Yaml inconsistent with the data added via ```bindingtree->addPropTree(ele.queryName(), LINK(&ele));```

Replacing all of it with a simple call to mergePTree() would be most consistent.  But of course we can discuss any issues that might have.